### ISSUE TITLE
[BUG] Sage Tabs button missing reference to pane

### ISSUE THREAD

**Issue by monkeypox8** [Kajabi/sage-lib#969]
## Detailed description
SageTab button controls output a boolean for `aria-controls` rather than the `id` of the [target tab pane as intended](https://www.digitala11y.com/aria-controls-properties/). Functionality is not impaired because `data-js-tabs-target` uses the correct reference. 

This only applies to the Rails component. The React version makes use of the correct attributes.

### Solution
In `docs/lib/sage_rails/app/views/sage_components/_sage_tab.html.erb`, remove `.present?` from `aria-controls="<%= component.target.present? %>"`

## Screenshots
<details>
<summary>Tabs example in Sage docs site</summary>
<img src="https://user-images.githubusercontent.com/816579/139339855-aa283d2a-2c1f-4051-be0b-b73d71cabb42.png" alt="">
</details>
